### PR TITLE
chore(web-modeler): update to version 0.3.0-beta

### PR DESCRIPTION
**Pull request by jfriedenstab**
## What is the purpose of the change
Update the docs for the Web Modeler Self-Managed 0.3.0-beta release (update Docker image tags, supported environments, third-party libraries).

## Are there related marketing activities
no

## When should this change go live?
once https://github.com/camunda/camunda-platform/pull/74 has been merged ✅

_Does this change need to be released before a certain date or milestone? If so, when?_
no

## PR Checklist
- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ We highly recommend running Camunda Platform 8 Self-Managed in a Kubernetes envi
...
+| Component                | Java version | Other requirements                                                                               |
+| ------------------------ | ------------ | ------------------------------------------------------------------------------------------------ |
+| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x (only if Elastic exporter is used)                                          |
+| Operate                  | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Tasklist                 | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Identity                 | OpenJDK 17+  | Keycloak 16.1.1, 19.0.3                                                                          |
+| Optimize                 | OpenJDK 11+  | Elasticsearch 7.13.x - 7.15.x, 7.16.2+, 7.17.x                                                   |
+| Web Modeler (Beta)       | -            | Keycloak 16.1.1, 19.0.3<br/>PostgreSQL 14.x (other database systems are currently not supported) |
```
Comment (pihme) [inline docs/reference/supported-environments.md:42]: :x: Earlier we had an issue with Keycload 16.1.1 https://github.com/camunda-cloud/identity/issues/1413

@Ben-Sheppard Did you fix the issue for 16.1.1? I was unsure after looking at your fix whether it also solves the issue for 16.1.1

File: versioned_docs/version-8.1/reference/supported-environments.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ We highly recommend running Camunda Platform 8 Self-Managed in a Kubernetes envi
...
+| Component                | Java version | Other requirements                                                                               |
+| ------------------------ | ------------ | ------------------------------------------------------------------------------------------------ |
+| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x (only if Elastic exporter is used)                                          |
+| Operate                  | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Tasklist                 | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Identity                 | OpenJDK 17+  | Keycloak 16.1.1, 19.0.3                                                                          |
+| Optimize                 | OpenJDK 11+  | Elasticsearch 7.13.x - 7.15.x, 7.16.2+, 7.17.x                                                   |
+| Web Modeler (Beta)       | -            | Keycloak 16.1.1, 19.0.3<br/>PostgreSQL 14.x (other database systems are currently not supported) |
```
Comment (pihme) [inline versioned_docs/version-8.1/reference/supported-environments.md:42]: See other comment

**Review by pihme — CHANGES_REQUESTED** [review]
Looks good except for the Keycloak 16.1.1 part

File: docs/reference/supported-environments.md (hunk 1 above)
Comment (jfriedenstab) [reply docs/reference/supported-environments.md:42]: I just copied the values from Identity that also states compatibility with both Keycloak versions. And isn't it rather an Identity issue anyway (if you can't copy a secret from their UI)? The Web Modeler itself is definitely compatible with Keycloak 16.1.1 and 19.0.3 :).

Comment (pihme) [reply docs/reference/supported-environments.md:42]: :smile:  If someone wants to test Web Modeler self managed, and it doesn't work (for whatever reason), we will be called :upside_down_face: 

But ok, seems like Ben is already on holidays. So feel free to merge it :santa:

**Review by pihme — APPROVED** [review]

File: docs/reference/supported-environments.md (hunk 1 above)
Comment (jfriedenstab) [reply docs/reference/supported-environments.md:42]: By the way, the latest stable version of Identity (`8.1.5`) – which is also used in the Docker Compose file for Self-Managed – does not support M2M applications yet anyway, so users won't get the chance to run into the issue 🙃.

Comment (pihme) [reply docs/reference/supported-environments.md:42]: Ah, I forgot the release cycle :see_no_evil:

**Comment by jfriedenstab** [comment]
@pepopowitz: I noticed your [comment here](https://github.com/camunda/camunda-platform-docs/pull/1581#issuecomment-1358404658) and was wondering why the `enforce-label` check is not running on this PR. It's stuck in "Expected — Waiting for status to be reported ". The branch is rebased on `main`. Is there anything else that needs to be done so this PR can get merged?

**Comment by christinaausley** [comment]
Just had to add a label here 👍

**Pull request merged**